plugin/xcodebuild.vim

```diff
@@ -35,11 +53,28 @@ function! s:project_file()
 endfunction
 
 function! s:scheme()
+  return '-scheme '. s:scheme_name()
+endfunction
+
+function!s:scheme_name()
   if !exists('s:chosen_scheme')
-    let s:chosen_scheme = system('source ' . s:plugin_path . '/bin/find_scheme.sh "' . s:project_file() . '"')
+    let s:chosen_scheme = system('source ' . s:bin_script('find_scheme.sh') . s:cli_args(s:project_file()))
+  endif
+
+  return s:chosen_scheme
+endfunction
+
+function! s:sdk()
+  if !exists('s:use_simulator')
+    let _ = system('source ' . s:bin_script('use_simulator.sh') . s:cli_args(s:project_file(), s:scheme_name()))
```

Comment:
What if you replace `let _ = system(...)` with `call system(...)`?

In variable assignment, `call` isn't necessary to call functions, but when calling them by themselves, it is. Maybe that's what's happening here?